flutter/pubspec.yaml

```diff
@@ -4,14 +4,14 @@ description: >-
 homepage: https://github.com/DavBfr/dart_barcode/tree/master/flutter
 repository: https://github.com/DavBfr/dart_barcode
 issue_tracker: https://github.com/DavBfr/dart_barcode/issues
-version: 2.0.4
+version: 2.0.5
 
 environment:
   sdk: ">=2.12.0 <4.0.0"
   flutter: ">=1.16.0"
 
 dependencies:
-  barcode: ^2.2.4
+  barcode: ^2.2.5
```

Comment:
You don't need to change this file. Pub will take the latest version

Reply:
I have removed this part of the edit in [b286841](https://github.com/DavBfr/dart_barcode/pull/65/commits/b2868410d5f09fdc5ec53684aa63439cdd4d29b7) .